compiler/z/codegen/OMRMachine.cpp

```diff
@@ -5327,40 +5327,58 @@ OMR::Z::Machine::coerceRegisterAssignment(TR::Instruction
          }
       else
          {
-         self()->cg()->traceRegAssigned(currentTargetVirtual, spareReg);
-
-         // virtual register is not assigned yet, copy register
-         cursor = self()->registerCopy(currentInstruction, currentTargetVirtualRK, targetRegister, spareReg, self()->cg(), instFlags);
-
-         spareReg->setState(TR::RealRegister::Assigned);
-         spareReg->setAssignedRegister(currentTargetVirtual);
-         currentTargetVirtual->setAssignedRegister(spareReg);
-
-         if (enableHighWordRA && currentTargetVirtual->is64BitReg())
+         if(spareReg == NULL)
```

Comment:
nit.  Missing a space between `if` and `(`.   This code to handle the `spareReg == NULL` case probably deserves a comment.

Reply:
Space fixed here: 
bb2f4d9

Comment: 
35913c9d6f78176b5617110005966b9613d4efdd